### DISCUSSION TITLE
Bluetooth mouse not working on iOS

### DISCUSSION THREAD

**Discussion by KesirVG** [ppy/osu#30361] [Q&A]
Hi i have a problem with my bt mouse not work with osu laser but my usb keyboard has work how to use mouse in osu laser?

**Comment by Joehuu** [comment]
Hmm.. works on my end, but the high precision mouse setting doesn't seem to remove mouse acceleration. And Bluetooth latency affects the experience as a whole.

Are you able to test with a USB mouse?
